dev/conditions.h

```diff
@@ -1015,21 +1035,49 @@ namespace sqlite_orm {
         return {std::move(o)};
     }
 
+#ifdef SQLITE_ORM_WITH_CPP20_ALIASES
+    template<orm_recordset_alias auto alias, class On>
```

Comment:
nit for the future: there is no requirement of making template names longer than one symbols, only argument/variable/constant names

Reply:
Understood. Though TBH I find it also nicer to read when template variables are spelled out, too.

Reply:
agree. It is up to you